.travis.yml

```diff
@@ -57,9 +62,9 @@ script:
 - if [[ ( "$TRAVIS_OS_NAME" == "linux" ) ]]; then gradle test -b test/java/build.gradle; fi
 - ./check-coverage.sh 80
 - npm test --prefix sysl2/sysl/sysl_js
-- GOOS=darwin GOARCH=amd64 go build -o gosysl/gosysl-darwin -ldflags "-X main.Version=${TRAVIS_TAG:1}" github.com/anz-bank/sysl/sysl2/sysl
-- GOOS=linux GOARCH=amd64 go build -o gosysl/gosysl-linux -ldflags "-X main.Version=${TRAVIS_TAG:1}" github.com/anz-bank/sysl/sysl2/sysl
-- go install -ldflags "-X main.Version=${TRAVIS_TAG:1}" github.com/anz-bank/sysl/sysl2/sysl
+- GOOS=${GOOS_DARWIN} GOARCH=${GOARCH} go build -o gosysl/gosysl-darwin -ldflags "-X \"main.Version=${TRAVIS_TAG:1}\" -X \"main.GitCommit=${TRAVIS_COMMIT}\" -X \"main.BuildDate=${BUILD_DATE}\" -X \"main.GoArch=${GOARCH}\" -X \"main.GoOS=${GOOS_DARWIN}\" -X \"main.GoVersion=${GO_VERSION}\" -X \"main.TravisBuildNum=${TRAVIS_BUILD_NUMBER}\" -X \"main.TravisDist=${TRAVIS_DIST}\" -X \"main.TravisOS=${TRAVIS_OS_NAME}\"" -v github.com/anz-bank/sysl/sysl2/sysl
+- GOOS=${GOOS_LINUX}  GOARCH=${GOARCH} go build -o gosysl/gosysl-linux  -ldflags "-X \"main.Version=${TRAVIS_TAG:1}\" -X \"main.GitCommit=${TRAVIS_COMMIT}\" -X \"main.BuildDate=${BUILD_DATE}\" -X \"main.GoArch=${GOARCH}\" -X \"main.GoOS=${GOOS_LINUX}\"  -X \"main.GoVersion=${GO_VERSION}\" -X \"main.TravisBuildNum=${TRAVIS_BUILD_NUMBER}\" -X \"main.TravisDist=${TRAVIS_DIST}\" -X \"main.TravisOS=${TRAVIS_OS_NAME}\"" -v github.com/anz-bank/sysl/sysl2/sysl
```

Comment:
This should be a script, and the envvars should just be parameters to the script.

Reply:
That is, there should be no env vars, just parameters.